examples/python/graphs/README.md

```diff
@@ -1,29 +1,51 @@
 <!--[metadata]
 title = "Graphs"
-tags = ["Graph", "Layout"]
-thumbnail = "https://static.rerun.io/graphs/19e6ca2e0752cdc7107c10b0cb79a4b7192d9e0b/480w.png"
-thumbnail_dimensions = [480, 399]
+tags = ["Graph", "Layout", "Node-link diagrams", "Bubble charts"]
+thumbnail = "https://static.rerun.io/graphs/1b93c00867821ebf3286653f43a9e5eb993f59ff/480w.png"
+thumbnail_dimensions = [480, 466]
```

Comment:
We usually always have exact square thumbnail for that. They get vertically cropped in the viewer anyways, so it's fine there, but this induces possible unwanted lateral cropping/rescaling on the landing page:

<img width="280" alt="image" src="https://github.com/user-attachments/assets/91573529-2da4-43f2-825a-82a0780af73f" />

Reply:
Also fyi we usually prepare/leave thumbnail in figma there: https://www.figma.com/design/U85f12k8Tw2ArD4N61cdtQ/Marketing-material?node-id=2467-22&t=j7o0AHPjhM3vRSB4-1

I wouldn't say that this is a requirement, but good to have it there if you pimp it a bit so we may iterate in the future.